notebooks/notebooks/iclr2022/win-rate-vs-training-steps.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    victim_change_steps = [
</pre><p>Why is this hardcoded?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>